src/LibYear/App.cs

```diff
@@ -18,16 +18,22 @@ public App(IPackageVersionChecker checker, IProjectFileManager projectFileManage
 
 	public async Task<int> Run(Settings settings)
 	{
+
 		_console.WriteLine();
 		var projects = await _projectFileManager.GetAllProjects(settings.Paths, settings.Recursive);
 		if (projects.Count == 0)
 		{
 			_console.WriteLine("No project files found");
 			return 1;
 		}
-
 		var result = await _checker.GetPackages(projects);
-		DisplayAllResultsTables(result, settings.QuietMode);
+		IOutput output = settings.Output switch
+		{
+			OutputOption.Console => new ConsoleOutput(_console),
+			OutputOption.Json => new JsonOutput(_console),
+			_ => throw new NotImplementedException()
+		};
```

Comment:
Can we pull this out into its own `GetOutput` expression-body method, and implicitly declare type with `var`?

Reply:
Yep, I like it.